pkg/version/version.go

```diff
@@ -24,4 +25,26 @@ func GetRuntimeVersion() string {
 		return "n/a\n"
 	}
 	return string(out)
+
+}
+
+// GetCLIVersion returns the version for the Dapr cli.
+func GetCLIVersion() string {
+	cliName := ""
```

Comment:
I think you just need to use `RootCmd.Version`. This is what returns the `dapr --version`. You already have it inside your code..

Reply:
It's causing a cyclic dependency when I tried that.